src/button/Button.ts

```diff
@@ -134,7 +127,7 @@ export class ButtonBase<P extends ButtonProperties = ButtonProperties> extends T
 			'aria-pressed': typeof pressed === 'boolean' ? pressed.toString() : null
 		}, [
 			...this.getContent(),
-			popup ? this.renderPopupIcon() : null
+			popup ? w(Icon, { extraClasses: { root: css.addon }, type: 'downIcon' }) : null
```

Comment:
this won't work as `extraClasses` is not themeable. You'll likely need to wrap it in a `span` or a `div` and pass the `css.addon` class to that instead.

Reply:
Wrapped them in a `span`